src/dirs.proj

```diff
@@ -58,7 +58,11 @@
     </TraversalBuildDependsOn>
   </PropertyGroup>
 
-
-
+  <!-- Workaround to remove items from PackagesNuSpecFiles (defined in packages.targets) -->
```

Comment:
Added this to stop trying to create packages those projects on the NetCore build.